### ISSUE TITLE
Will this be an issue?

### ISSUE THREAD

**Issue by COD1995** [safreita1/malnet-graph#6]
In the model forward function: 

return `F.log_softmax(x, dim=-1)` , however in training loop loss is `loss = F.cross_entropy(output, data.y)` ; This combination create a mathematical issue:

if you use log_softmax you are returning log probabilities, but per pytorch documentation https://docs.pytorch.org/docs/stable/generated/torch.nn.functional.cross_entropy.html cross_entropy requires raw logits not probabilities. 

This will sometimes cause training stability issues as cross_entropy function also apply log_softmax to it; double softmax will cause the sum of total probability of not equal to 1!!
